source/gameengine/Ketsji/KX_LodManager.cpp

```diff
@@ -52,78 +51,49 @@ KX_LodList::KX_LodList(Object *ob, KX_Scene* scene, KX_BlenderSceneConverter* co
 			}
 
 			if (lod->flags & OB_LOD_USE_HYST) {
-				lodLevel.flags |= Level::USE_HYST;
+				lodLevel->m_flags |= KX_LodLevel::USE_HYST;
 			}
 
-			lodLevel.level = level++;
-			lodLevel.meshobj = BL_ConvertMesh(lodmesh, lodmatob, scene, converter, libloading);
-			lodLevel.distance = lod->distance;
-			lodLevel.hysteresis = lod->obhysteresis;
+			lodLevel->m_level = level++;
```

Comment:
I think you can use a constructor since `KX_LodLevel` is not anymore a struct. You just have to define a `int flag` 5 lines above. The constructor will look like:
`KX_LodLevel(level, meshobj, distance, hysteresis, flag)`